docs/guide/api-javascript.md

```diff
@@ -44,6 +44,14 @@ The `InlineConfig` interface extends `UserConfig` with additional properties:
 - `configFile`: specify config file to use. If not set, Vite will try to automatically resolve one from project root. Set to `false` to disable auto resolving.
 - `envFile`: Set to `false` to disable `.env` files.
 
+## `ResolvedConfig`
+
+The `ResolvedConfig` interface has all the same properties of a `UserConfig`, except most properties are resolved and non-undefined. It also contains utilities like:
+
+- `config.createResolver`: Create a function to resolve `id`s using core Vite plugins.
```

Comment:
I would prefer we remove this one from the PR, so we can discuss documenting it in a separate PR. We could mark it as deprecated also during v3 and offer one where we return the full resolved object instead of the `.id`

Reply:
Alright yeah let's hide this for now until we figure out a path forward for it.